File: editorial/chap06/python/answer_B43.py
# 입력
N, M = map(int, input().split())
A = list(map(int, input().split()))

# 배열 준비
Incorrect = [ 0 ] * (N + 1)

# 오답 수 구하기
for i in range(M):
	Incorrect[A[i]] += 1

# 답을 출력
for i in range(1, N+1):
	print(M - Incorrect[i])

File: editorial/chap06/cpp/answer_B43.cpp
#include <iostream>
using namespace std;
 
int N, M;
int A[200009];
int Incorrect[200009];
 
int main() {
	// 입력
	cin >> N >> M;
	for (int i = 1; i <= M; i++) cin >> A[i];
 
	// 오답 수 구하기
	for (int i = 1; i <= N; i++) Incorrect[i] = 0;
	for (int i = 1; i <= M; i++) Incorrect[A[i]] += 1;
 
	// 답을 출력
	for (int i = 1; i <= N; i++) cout << M - Incorrect[i] << endl;
	return 0;
}
